views/scss/comments-panel.scss

```
@@ -174,17 +217,19 @@
         display: block;
-        font-family: 'Source Sans Pro', sans-serif;
-        font-size: 12px;
-        font-weight: 700;
-        line-height: 18px;
-        height: 18px;
-        letter-spacing: 0.5px;
+        width: 3px;
+        height: 3px;
+        margin: 0;
+        border-radius: 50%;
+        background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)